### ISSUE TITLE
feature request: add sega model 3 core (supermodel)

### ISSUE THREAD

**Issue by ghost** [libretro/RetroArch#17084]
hi

I want to request adding  sega model 3 core (supermodel)

https://www.supermodel3.com/

https://github.com/trzy/Supermodel

Thanks!

**Comment by OctopusButtons** [comment]
Yes that would be awesome.  Supermodel is amazing and great work by the devs, but somewhat cumbersome to use because of CLI, and also the last time I tried it with ReShade CRT shaders that was making it crash :(

**Comment by gouchi** [comment]
For the initial work, please check this [repository](https://github.com/libretro/supermodel).
